modules/fedimint-mint-client/src/client_db.rs

```diff
@@ -13,6 +14,7 @@ pub enum DbKeyPrefix {
     Note = 0x20,
     NextECashNoteIndex = 0x2a,
     CancelledOOBSpend = 0x2b,
+    RestoreState = 0x2c,
```

Comment:
It had to happen eventually (in fact, it already did), still sad that we are now overloading the DB key prefixes. It's not a problem due to DB isolation, but reading raw hex encoded data will be a bit more ambiguous (e.g. if one doesn't know the instance<>kind mapping).

Reply:
Yeah, we'll need dbdump do a good job. eg. even for keys that are not implemented print `<kind> <prefix>` or something.